backend/README.md

```diff
@@ -27,7 +27,7 @@ Linting is done using ruff, formatting using black and type checking using mypy.
 
 ### 4. Start the development environment
 
-Assuming that your microk8s cluster is running, you can start the development environment by simply running `$ skaffold dev`. This command will build the docker images and push them to your microk8s registry, then apply your k8s manifest to start the cluster and pull those images. Additionally, skaffold will watch for file changes and either sync them directly inside the running containers or rebuild and redeploy k8s cluster for you automatically.
+Assuming that your microk8s cluster is running, you can start the development environment by simply running `$ skaffold dev --no-prune=false --cache-artifacts=false`. This command will build the docker images and push them to your microk8s registry, then apply your k8s manifest to start the cluster and pull those images. Additionally, skaffold will watch for file changes and either sync them directly inside the running containers or rebuild and redeploy k8s cluster for you automatically.
```

Comment:
It's a shame `skaffold` spells it as `artifacts` :smile:

Reply:
actually that makes it easier to make the distinction between our artefacts and everyone else's artifacts :smiley: